app/lib/company_pages_analytics.rb

```diff
@@ -0,0 +1,8 @@
+class CompanyPagesAnalytics
```

Comment:
a whole new class only for this count?

Reply:
yes :) I've asked it. It will get more methods in future